commit-event-service/src/test/scala/io/renku/commiteventservice/events/categories/globalcommitsync/eventgeneration/gitlab/GitLabCommitStatFetcherSpec.scala

```diff
@@ -119,11 +122,29 @@ class GitLabCommitStatFetcherSpec
     implicit val maybeAccessToken: Option[AccessToken] = personalAccessTokens.generateSome
     val projectId = projectIds.generateOne
 
-    val gitLabUrl           = GitLabUrl(externalServiceBaseUrl)
     val gitLabCommitFetcher = mock[GitLabCommitFetcher[IO]]
     private implicit val logger: TestLogger[IO] = TestLogger[IO]()
+    val gitLabClient = mock[GitLabClient[IO]]
     val gitLabCommitStatFetcher =
-      new GitLabCommitStatFetcherImpl[IO](gitLabCommitFetcher, gitLabUrl.apiV4, Throttler.noThrottling)
+      new GitLabCommitStatFetcherImpl[IO](gitLabCommitFetcher, gitLabClient)
+
+    val uri = uri"projects" / projectId.show withQueryParams Map("statistics" -> true)
+    val endpointName: String Refined NonEmpty = "projects"
```

Comment:
Shouldn't the endpoint name be _project-details_ or smth similar as we are fetching a single entity here?